lib/elixir/lib/kernel.ex

```diff
@@ -3308,7 +3308,7 @@ defmodule Kernel do
   end
 
   defp nest_pop_in(:map, h, [{:access, key}]) do
-    quote do
+    quote generated: true do
```

Comment:
Can you annotate the one a couple lines below too? It is just a matter of time until it warns too.

Reply:
Done [e165702](https://github.com/elixir-lang/elixir/pull/14064/commits/e16570202b16b75dc5084c5ae04ca92de1a6f16a)